pkg/inject/inject.go

```diff
@@ -55,7 +55,8 @@ type ResourceConfig struct {
 	nsAnnotations         map[string]string
 	meta                  metav1.TypeMeta
 	obj                   runtime.Object
-	objMeta               objMeta
+	objMeta               *metav1.ObjectMeta
+	templateMeta          objMeta
```

Comment:
i'm struggling to parse a field named `objMeta` and another field of type `objMeta`. mind naming them a bit more explicitly? maybe something like:
```go
workLoadMeta  *metav1.ObjectMeta
podMeta       objMeta
```

Reply:
You're right, that's confusing. I'll apply your suggestion :+1: